Project/ProjectSettings/DynamicsManager.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!55 &1
 PhysicsManager:
   m_ObjectHideFlags: 0
-  serializedVersion: 7
+  serializedVersion: 10
```

Comment:
Just curious this changed in the physics manager?

Reply:
I'm not sure what this change to the physics manager is. i didn't intentionally change any of the physics settings that i know of.